test/src/e2e/mempool.test.ts

```diff
@@ -46,26 +47,54 @@ describe("Sealing test", function() {
 });
 
 describe("Future queue", function() {
+        let node: CodeChain;
+
+        beforeEach(async function() {
+            node = new CodeChain();
+            await node.start();
+        });
+
+        it("all pending transactions must be mined", async function() {
+            const seq = (await node.sdk.rpc.chain.getSeq(faucetAddress)) || 0;
+            await node.sendPayTx({ seq: seq + 3 });
+            expect(await node.sdk.rpc.chain.getSeq(faucetAddress)).to.equal(seq);
+            await node.sendPayTx({ seq: seq + 2 });
+            expect(await node.sdk.rpc.chain.getSeq(faucetAddress)).to.equal(seq);
+            await node.sendPayTx({ seq: seq + 1 });
+            expect(await node.sdk.rpc.chain.getSeq(faucetAddress)).to.equal(seq);
+            await node.sendPayTx({ seq: seq });
+            expect(await node.sdk.rpc.chain.getSeq(faucetAddress)).to.equal(
+                seq + 4
+            );
+        });
+
+    afterEach(async function() {
+        if (this.currentTest!.state === "failed") {
+            node.keepLogs();
+        }
+        await node.clean();
+    });
+});
+
+describe("Get Pending Transaction", function() {
     let node: CodeChain;
 
     beforeEach(async function() {
         node = new CodeChain();
         await node.start();
     });
 
-    it("all pending transactions must be mined", async function() {
-        const seq = (await node.sdk.rpc.chain.getSeq(faucetAddress)) || 0;
-
-        await node.sendPayTx({ seq: seq + 3 });
-        expect(await node.sdk.rpc.chain.getSeq(faucetAddress)).to.equal(seq);
-        await node.sendPayTx({ seq: seq + 2 });
-        expect(await node.sdk.rpc.chain.getSeq(faucetAddress)).to.equal(seq);
-        await node.sendPayTx({ seq: seq + 1 });
-        expect(await node.sdk.rpc.chain.getSeq(faucetAddress)).to.equal(seq);
-        await node.sendPayTx({ seq: seq });
-        expect(await node.sdk.rpc.chain.getSeq(faucetAddress)).to.equal(
-            seq + 4
-        );
+    it.only("all transaction in both queues should be included", async function() {
```

Comment:
You need to remove `only` here.

Reply:
Plus, in this case you need to add a new test,  not changing the existing one.

Reply:
Done ✅